src/core/EdFi.DataManagementService.Core/Configuration/AppSettings.cs

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DataManagementService.Core.Configuration
+{
+    public class AppSettings
+    {
+        public int BeginAllowedSchoolYear { get; set; }
+        public int EndAllowedSchoolYear { get; set; }
```

Comment:
Funny thing here: we don't need these. They were accidentally imported from Meadowlark. Can you go ahead and remove these two values everywhere?

I'm pretty confident that we _will_ have other settings to place here in the future.